dockerfiles/2.7.1-migration/data/home/pootle/.pootle/pootle.conf

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Sample configuration file.
+
+This file includes the settings that administrators will likely change.
+You can find the defaults in the ``*.conf`` files for more advanced settings.
+
+In order for this configuration changes to take effect, bear in mind that you
+need to change the extension of this file from ``.conf.sample`` to ``.conf``.
+"""
+
+
+#
+# Base
+#
+
+# Local time zone for this installation. Choices can be found here:
+# http://en.wikipedia.org/wiki/List_of_tz_zones_by_name
+# although not all choices may be available on all operating systems.
+# If running in a Windows environment this must be set to the same as your
+# system time zone.
+TIME_ZONE = 'UTC'
+
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = 'TSMAV+BXBD5WkYrzwzZWLCVQa2dTawAjkEY/8J4PuW20qh6Ri2wRlU0AaCSlaDZSzGw='
+
+# A list of strings representing the host/domain names that this Pootle server
+# can serve. This is a Django's security measure. More details at
+# https://docs.djangoproject.com/en/dev/ref/settings/#allowed-hosts
+ALLOWED_HOSTS = ['127.0.0.1', 'localhost']
+
+#
+# Backends
+#
+
+# Database backend settings
+DATABASES = {
+    'default': {
+        # Replace 'sqlite3' with 'postgresql_psycopg2', 'mysql' or 'oracle'.
+        'ENGINE': 'django.db.backends.sqlite3',
```

Comment:
no - beacause this starts as a 2.5.1 install

transaction hooks are injected during upgrade